sdk/communication/communication-phone-numbers/test/public/siprouting/setTrunks.spec.ts

```diff
@@ -28,59 +42,73 @@ matrix([[true, false]], async function (useAad) {
       }
     });
 
-    it("can set a new trunk", async () => {
-      const trunk: SipTrunk = { fqdn: "111.fqdn.com", sipSignalingPort: 1231 };
+    it("can set a new trunk", async function (this: Context) {
+      if (isPlaybackMode()) {
```

Comment:
if we're skipping it in playback, should we skip it in record too? Would a better check be only running it in live?

Reply:
Makes sense. 
It is not needed in current solution with recorder.variable API anymore. Removed.